dev/system-test/firestore.ts

```diff
@@ -5010,6 +5010,16 @@ describe('Aggregation queries', () => {
     return Promise.all(sets);
   }
 
+  it('can run count within a transaction with readtime', async () => {
+    const doc = col.doc();
+    const writeResult: WriteResult = await doc.create({some: 'data'});
+    const count = await firestore.runTransaction(t => t.get(col.count()), {
+      readOnly: true,
+      readTime: writeResult.writeTime,
+    });
+    expect(count.data().count).to.equal(1);
```

Comment:
Would it make sense to run two transactions, one with a readtime before the write and one with a readtime after?

Reply:
I expanded the test. PTAL

Reply:
LGTM. thanks!